src/main/resources/init_db.sql

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE `taxi_db`;
```

Comment:
It would be better if you use 'CREATE SCHEMA IF NOT EXISTS; instead of only 'CREATE DATABASE'

Reply:
CREATE DATABASE creates a whole new entire database structure with its own catalog tables etc.

CREATE SCHEMA creates a new namespace for objects within an existing database in systems like Oracle that only support a single database in a server instance. Some multi-database systems support CREATE SCHEMA within each database. A single set of database catalog tables contain the multiple schema definitions.

Others, Informix is one of these, treat CREATE SCHEMA simply as a way to group multiple DDL statements into a single logical unit. The statements contained within the CREATE SCHEMA … block are not processed differently than if they were issued standalone.